schemes/musica/tuvx/musica_ccpp_tuvx_gas_species_profile.F90

```diff
@@ -0,0 +1,219 @@
+module musica_ccpp_tuvx_gas_species_profile
+  use ccpp_kinds, only: kind_phys
+  use musica_tuvx_profile, only: profile_t
+
+  implicit none
+
+  private
+  public :: create_gas_species, configure_gas_species, create_gas_species_profile_group, &
+            set_gas_species_values, deallocate_gas_species, deallocate_gas_species_profile_group
+
+  !> Conversion factor from km to cm
+  real(kind_phys), parameter, public :: km_to_cm = 1.0e5
+  !> Conversion factor from m3 to cm3
+  real(kind_phys), parameter, public :: m_3_to_cm_3 = 1.0e6
+  !> Molar mass value of dry air is obtained from 'CAM-SIMA/src/utils/std_atm_profile.F90'
+  ! TODO(jiwon)
+  real(kind_phys), parameter, public :: MOLAR_MASS_DRY_AIR = 0.0289644_kind_phys ! kg mol-1
```

Comment:
Hi @mattldawson and @nusbaume,  I get the molar mass value of dry air from [CAM-SIMA/src/utils/std_atm_profile.F90](https://github.com/ESCOMP/CAM-SIMA/blob/development/src/utils/std_atm_profile.F90#L48). Is it okay to define it here?  We discussed about passing parameters from CAM-SIMA, but I'm still unsure about the approach we're going to take.

Reply:
Hi @boulderdaze, you should be able to read in the molar mass of dry air as an input argument with the standard name `molecular_weight_of_dry_air`, which will be in units of `g mol-1`.  There is also a composition-dependent version as well, but that is only used in WACCM and WACCM-X configurations (although if you would like to use that quantity instead just let me know).

The reason we generally suggest passing this value in as an argument is because some host models and host model configurations might have slightly different values for the dry air molar mass, especially if they are configured for high-top, paleoclimate, or exoplanet simulations, and so by bringing it in as an argument we won't have to worry about molar mass value mismatches.

Anyways, hopefully that helps?  If for some reason that method does not work for your particular setup here just let me know.  Thanks!

Reply:
Ah that makes sense, thank you for the additional details! I will use `molecular_weight_of_dry_air` as an input.